frontend/src/component/feedbackNew/FeedbackProvider.tsx

```diff
@@ -0,0 +1,43 @@
+import { FeedbackComponent } from './FeedbackComponent';
+import { FeedbackContext } from './FeedbackContext';
+import { ReactNode, useState } from 'react';
+import { ProvideFeedbackSchema } from '../../openapi';
+
+interface IFeedbackProviderProps {
+    children: ReactNode;
```

Comment:
I think you can skip it if you type your component as FC since it's a React type